lambdas/README.md

```diff
@@ -4,22 +4,68 @@ Serverless configured lambdas running on AWS.
 
 ---
 
-## Meetup.com
 
-Lambdas used to get and process events from Meetup.com. There are helper scripts
-to aid in deployment and development. Use `npm run <command>` with any of the
-below.
+### First time setup
+
+#### Create a new AWS Profile within `~/.aws/credentials`
 
-For initial setup, please create a `meetupcomEvents` profile in your
+For initial setup, please create a `muxer` profile in your
 `~/.aws/credentials` file. Serverless will use this when running the commands
 listed below.
 
 ```
-[meetupcomEvents]
+[muxer]
```

Comment:
My reasoning here was that we could avoid the overhead of managing individual lambda profiles and use a single default profile. As the keys are the same, and we don't make use of different regions a single profile saved in setup time. Does this sound reasonable @alistairjcbrown?

Reply:
Makes sense to me - IIRC, I was playing it safe, working with a new tech and making sure I didn't accidentally couple them together. But logistically, this makes more sense 👍